pkg/config/setup/system_probe.go

```diff
@@ -43,6 +43,11 @@ const (
 	// defaultBTFOutputDir is the default path for extracted BTF
 	defaultBTFOutputDir = "/var/tmp/datadog-agent/system-probe/btf"
 
+	// defaultDynamicInstrumentationDebugInfoDir is the default path for debug
+	// info for dynamic instrumentation this is the directory where the debug
+	// info is decompressed into during processing.
+	defaultDynamicInstrumentationDebugInfoDir = "/var/tmp/datadog-agent/system-probe/dynamic-instrumentation/decompressed-debug-info"
```

Comment:
What about Windows ?

Reply:
This feature doesn't work on windows and the consumer of this setting has a build tag for `linux_bpf`, but I'm open to suggestions if there's something more appropriate to use here.